src/main/java/io/github/cottonmc/resources/CottonResources.java

```diff
@@ -68,7 +68,7 @@
 	 */
 	public static final Map<String, ResourceType> BUILTINS = new HashMap<>();
 
-	public static ItemGroup ITEM_GROUP = FabricItemGroupBuilder.build(new Identifier(MODID, "resources"), () -> new ItemStack(BUILTINS.get("copper").getItem("gear")));
+	public static ItemGroup ITEM_GROUP = FabricItemGroupBuilder.build(new Identifier(MODID, "resources"), () -> new ItemStack(BuiltinResources.COPPER.getGear().get()));
```

Comment:
Would Java complain about `Optional#get` being called without checking `Optional#isPresent` first to a level that would interrupt compilation?

Reply:
Or do `orElseThrow(IllegalStateException::new)`